lib/index.js

```diff
@@ -9,7 +9,8 @@ $(document).ready(() => {
   })
   $('#food-table-info').on('click', event => {
     if (event.target.nodeName == 'BUTTON') {
-      foodsRequests.deleteFood(event.target.id)
+      var id = event.target.id.split('-')[2]
```

Comment:
This change has already been added and committed to master https://github.com/izaxveta/quantified-self/commit/2a30ad0176618ed6ef5897685d29cc754a1d4168

Did you pull the latest changes from master into this branch before pushing up?